rpcd/playbooks/roles/rpc_maas/tasks/agent_install_remote.yml

```diff
@@ -13,22 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Verify raxmon-entities-list and MaaS API authentication does not fail
-  shell: |
```

Comment:
I actually removed that because the next task (which also does a raxmon-entities-list) will now fail correctly due to the pipefail, so it seemed redundant.  Should it get added back in?